code/e2e-tests/framework-nextjs.spec.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable jest/no-disabled-tests */
+import type { Locator } from '@playwright/test';
+import { test, expect } from '@playwright/test';
+import process from 'process';
+import { SbPage } from './util';
+
+const storybookUrl = process.env.STORYBOOK_URL || 'http://localhost:6006';
+const templateName = process.env.STORYBOOK_TEMPLATE_NAME || 'nextjs/default-js';
```

Comment:
I'm not sure if we should have nextjs as default value, this could lead into misdirection.. maybe add a log if that is not set, so people running tests locally understand that things are not set?

```suggestion
const templateName = process.env.STORYBOOK_TEMPLATE_NAME;
```

Reply:
Good catch!